eng/Versions.props

```diff
@@ -6,18 +6,18 @@
 -->
 <Project>
   <PropertyGroup Label="Version settings">
-    <AspNetCoreMajorVersion>6</AspNetCoreMajorVersion>
+    <AspNetCoreMajorVersion>7</AspNetCoreMajorVersion>
     <AspNetCoreMinorVersion>0</AspNetCoreMinorVersion>
     <AspNetCorePatchVersion>0</AspNetCorePatchVersion>
-    <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
+    <PreReleaseVersionIteration>0</PreReleaseVersionIteration>
```

Comment:
I think this should be 1, though it shouldn't affect anything. We only brand as alpha initially to avoid any possible package precedence issues before we switch to previews.

Reply:
Just caught that too, fixed